VetClinic.API/Validators/Doctor/CreateDoctorDtoValidator .cs

```diff
@@ -0,0 +1,13 @@
+﻿using FluentValidation;
+using VetClinic.API.DTO.Doctor;
+
+namespace VetClinic.API.Validators.Doctor
+{
+    public class CreateDoctorDtoValidator : AbstractValidator<CreateDoctorDto>
+    {
+        public CreateDoctorDtoValidator()
+        {
+            
```

Comment:
Please add TO DO if you plan to implement this part of code later

Reply:
Wrote comment for this part.